CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 0.37.1 (July 23, 2025)
+
+ENHANCEMENTS:
+
+* feat(r/trigger): Accommodate the "dataset_slug" field in triggers (#720)
```

Comment:
This one is only for the client -- not the trigger resource. I called it a bug as it was an oversight (and then fits into the patch more cleanly) but I am happy to call it a feature and still ship it in this patch :)

```suggestion
* fix(client): Accommodate the "dataset_slug" field in triggers (#720)
```